packages/playwright/src/transform/compilationCache.ts

```diff
@@ -218,9 +218,15 @@ export function collectAffectedTestFiles(dependency: string, testFileCollector:
     if (deps.has(dependency))
       testFileCollector.add(testFile);
   }
-  for (const [testFile, deps] of externalDependencies) {
-    if (deps.has(dependency))
-      testFileCollector.add(testFile);
+  for (const [importingFile, deps] of externalDependencies) {
+    if (deps.has(dependency)) {
+      testFileCollector.add(importingFile);
+
+      for (const [testFile, deps] of fileDependencies) {
```

Comment:
nit: can we avoid shadowing the `deps` variable? I find it harder to read.

Reply:
done!